libbeat/scripts/Makefile

```diff
@@ -116,7 +117,8 @@ integration-tests-environment:
 .PHONY: system-tests
 system-tests: libbeat.test prepare-tests system-tests-setup
 	. build/system-tests/env/bin/activate; nosetests -w tests/system --process-timeout=$(TIMEOUT) --with-timer
-	python ../scripts/aggregate_coverage.py -o ${COVERAGE_DIR}/system.cov ./build/system-tests/run
+	python ${GOPATH}/src/github.com/elastic/beats/scripts/aggregate_coverage.py -o ${COVERAGE_DIR}/system.cov ./build/system-tests/run
```

Comment:
If I was writing a community beat and I vendored `github.com/elastic/beats` in my project, this path wouldn't be correct, would it (since `GOPATH` won't contain my vendor dir)?